get-started/document-agent's-general-description+purpose.md

```diff
@@ -0,0 +1,17 @@
+## Cryostat Agent's general description + purpose
+
+The Cryostat agent is a new component of Cryostat, an application used for retrieving data from Java applications that runs on the Java Virtual Machine (JVM). The Cryostat agent is a Java agent that acts as a plugin system for the applications running on the JVM. Prior to the agent, Cryostat used the JMX to extract data from the JVM by initiating a conversation with those JVMs. It then fetches the data from JFR and pulls it over the network back toward the Cryostat to be accessible to end users.
```

Comment:
I think the "agent" should always be capitalized like a proper noun:

"Cryostat agent" -> "Cryostat Agent"

There are a few more cases like this later in the doc, too.

Reply:
"the JMX" should just be "JMX".

I think this can be turned around as well:

"Prior to the agent, Cryostat used the JMX to extract data from the JVM by initiating a conversation with those JVMs" -> "Prior to the Agent, Cryostat extracted data from the JVM by initiating a conversation over JMX". Does that sound okay?